app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
Em tìm hiểu về `I18n in rails`, sau đó áp dụng vào các text của TOÀN BỘ file HTML nhé
` Ruby on Rails Tutorial Sample App`

Reply:
https://www.railstutorial.org/book/rails_flavored_ruby
cái này em thấy giống như ở phần đầu chapter 4 đó chị.

Reply:
Cái đó là khác nữa nhé

sau đó áp dụng vào các text của TOÀN BỘ file HTML nhé
`Ruby on Rails Tutorial Sample App`